### PR TITLE
Fix env variable names in doc to match install.sh

### PR DESCRIPTION
**Pull request by zombiezen**

### CODE REVIEW

**Comment by garply** [comment]
Hey admins! Can one of you let me know if it's safe to build this pull request? See https://alpha.sandstorm.io/grain/kB7bofATTL2jGuKSFwmqdA/GithubPRCommands for a list of commands.

**Comment by paulproteus** [comment]
Thanks! I'll review this & presumably merge & update docs on Monday!

File: docs/administering/install-script.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Over the time we have spent maintaining the install script, we hae found it easi
 user-provided configuration options from environment variables, rather than command line flags. Here
 are some environment variables that `install.sh` can look for, and their meanings.
 
-- `OVERRIDE_SANDSTORM_DEFAULT_INSTALL_DIR`: If you specify this, Sandstorm will install into this
+- `OVERRIDE_SANDSTORM_DEFAULT_DIR`: If you specify this, Sandstorm will install into this
```
Comment (paulproteus) [inline docs/administering/install-script.md]: +1, verified

File: docs/administering/install-script.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ free HTTPS certificate service.
 
 - `DESIRED_SANDCATS_NAME`: The name of the the sandcats.io subdomain you would like to use for this install.
 
-- `DOMAIN_RESERVATION_TOKEN`: A token that indicates you have pre-reserved a sandcats.io subdomain.
+- `SANDCATS_DOMAIN_RESERVATION_TOKEN`: A token that indicates you have pre-reserved a sandcats.io subdomain.
```
Comment (paulproteus) [inline docs/administering/install-script.md]: +1, verified

File: docs/administering/install-script.md [hunk 3]
```diff
@@ -99,7 +99,7 @@ To pass an environment variable to the Sandstorm installer, you can do as follow
 
 ```bash
 curl https://install.sandstorm.io/ > install.sh
-sudo OVERRIDE_SANDSTORM_DEFAULT_INSTALL_DIR=/opt/sandstorm-is-awesome bash install.sh -d
+sudo OVERRIDE_SANDSTORM_DEFAULT_DIR=/opt/sandstorm-is-awesome bash install.sh -d
```
Comment (paulproteus) [inline docs/administering/install-script.md]: +1, verified

**Comment by paulproteus** [comment]
I am very happy to merge this. Thanks @zombiezen for catching my mistakes!

**Comment by paulproteus** [comment]
@zombiezen your changes are live!

**Pull request merged**